src/Types/BigIntType.php

```diff
@@ -47,15 +44,15 @@ public function convertToPHPValue(mixed $value, AbstractPlatform $platform): int
             return $value;
         }
 
-        if ($value > PHP_INT_MIN && $value < PHP_INT_MAX) {
-            return (int) $value;
-        }
-
         assert(
             is_string($value),
             'DBAL assumes values outside of the integer range to be returned as string by the database driver.',
         );
 
+        if ($value === (string) (int) $value) {
+            return (int) $value;
```

Comment:
psalm is wrong here - https://github.com/vimeo/psalm/issues/10995

tell me how to fix the CI

Reply:
@greg0ire any advice how to fix the psalm CI?

Reply:
@greg0ire how should I make psalm happy when there is known bug in psalm?

Reply:
Ignoring the error is fine.